### PR TITLE
Cca charts

### PR DESCRIPTION
**Pull request by arunasd463**
**This PR includes:**

- CCA charts using d3js library.
- Added dynamic charts for home and groups.
- Added dynamic stats component in home page and groups.
- Navbar improvements.

### CODE REVIEW

File: src/components/@core/navbar/index.tsx [hunk 1]
```diff
@@ -18,10 +18,10 @@ export default function NavBar() {
         backdropFilter="saturate(180%) blur(5px)"
         bg="whiteAlpha.800"
         id="menu"
-        position="fixed"
         width="100%"
         zIndex={4}
         shadow="md"
+        position={"fixed"}
```
Comment (Logan1x) [inline src/components/@core/navbar/index.tsx:24]: How is `position="fixed"` and `position={"fixed"}` are different?

File: src/components/charts/hooks/use-tooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    .style("color", "white")
+    .style("font-size", "0.875rem")
+    .style("max-width", "320px")
+    .style("box-shadow", "0 4px 6px -1px rgba(0,0,0,0.1), 0 2px 4px -1px rgba(0,0,0,0.06)");
+};
+
+export function tooltipHelpers(tip, tooltipRenderer, leftOffset = 40, topOffset = 30) {
+  const mousemove = (e) => {
+    tip
+      .style("left", pointer(e)[0] + leftOffset + "px") // It is important to put the +90: other wise the tooltip is exactly where the point is an it creates a weird effect
```
Comment (Logan1x) [inline src/components/charts/hooks/use-tooltip.tsx:23]: is this still relevant? I cant see `+90:` anywhere.

File: src/components/pages/chart/index.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  const dataForChart = Object.entries(chartData.stats.satewiseAggregation)
+    .map(([state, count]) => ({
+      Name: state.toUpperCase(),
+      Value: count
+    }))
+    .filter((item) => item.Name !== "?")
+    .sort((a, b) => (b.Value as number) - (a.Value as number));
+
+  const stateData = dataForChart.length > 0 && {
+    Title: "Statewise distribution",
```
Comment (Logan1x) [inline src/components/pages/chart/index.tsx:24]: should we move this to translation file?

**Review by Logan1x — APPROVED** [review]
Other than minor changes/questions. The PR seems good to go.

File: src/components/@core/navbar/index.tsx (hunk 1 above)
Comment (arunasd463) [reply src/components/@core/navbar/index.tsx:24]: Both are same , we can use both.

File: src/components/pages/chart/index.tsx (hunk 3 above)
Comment (arunasd463) [reply src/components/pages/chart/index.tsx:24]: yes it can have translation.

**Pull request merged**